source/EngineInterface/DescriptionEditService.cpp

```diff
@@ -16,7 +16,7 @@ CollectionDescription DescriptionEditService::createRect(CreateRectParameters co
     CollectionDescription result;
     for (int i = 0; i < parameters._width; ++i) {
         for (int j = 0; j < parameters._height; ++j) {
-            result.addCell(CellDescription()
+            result.cells().push_back(CellDescription()
```

Comment:
Wrong, use `result._cells.emplace_back(CellDescription()..`
Change the other places in the same way.